### PR TITLE
curl 7.77.0

### PR DESCRIPTION
**Pull request by derrabus**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/curl.rb [hunk 1]
```diff
@@ -2,16 +2,10 @@ class Curl < Formula
...
-  revision 1
 
   stable do
-    url "https://curl.se/download/curl-7.76.1.tar.bz2"
-    sha256 "7a8e184d7d31312c4ebf6a8cb59cd757e61b2b2833a9ed4f9bf708066e7695e9"
-
-    patch do
-      url "https://github.com/curl/curl/commit/8bdde6b14ce3b5fd71c772a578fcbd4b6fa6df19.patch?full_index=1"
-      sha256 "4da7b91474583c563dcabb039405423dbaf0e1df92ce97bbe6726bcfdb67e602"
-    end
```
Comment (derrabus) [inline Formula/curl.rb:14]: I've removed the patch because the corresponding commit is included in cURL 7.77.0.

Comment (nandahkrishna) [reply Formula/curl.rb:14]: @derrabus could you remove the `url` and `sha256` from the `stable` block and place them below the `homepage` instead (like other formulae without the `stable` block)?

Comment (derrabus) [reply Formula/curl.rb:14]: Sure, done.

**Review by nandahkrishna — APPROVED** [review]
Thanks, @derrabus!

**Pull request closed without merging**